.travis.yml

```diff
@@ -40,16 +40,15 @@ before_install:
   - wget http://bit.ly/miniconda -O miniconda.sh
   - bash miniconda.sh -b -p $HOME/miniconda
   - export PATH="$HOME/miniconda/bin:$PATH"
-  - conda update conda --yes
+  - conda config --set always_yes yes --set changeps1 no --set show_channel_urls true
+  - conda update conda
   - conda config --set show_channel_urls true
   - conda config --add channels odm2 --force
-  - conda create --yes -n TEST python=$TRAVIS_PYTHON_VERSION --file requirements.txt --file requirements-dev.txt
+  - conda create -n TEST python=$TRAVIS_PYTHON_VERSION --file requirements.txt --file requirements-dev.txt
   - source activate TEST
   - if [[ "$TRAVIS_PYTHON_VERSION" == "2.7" ]]; then
-      conda install --yes configparser ;
+      conda install configparser ;
     fi
-  # Coding standards.
-  - conda install --channel conda-forge --yes flake8-print flake8-quotes flake8-import-order flake8-builtins flake8-comprehensions flake8-mutable
```

Comment:
By moving this to the test area we avoid `conda-forge` in the main tests and we get rid of the `libconv` issue altogether without any channel workaround.

Reply:
Ah cool! Interesting stuff! These packages are in `requirements-dev.txt` already.

Reply:
I guess they're not. Seems like you just moved them below.

Reply:
I never added them to the requirement-dev b/c they are coming from `conda-forge` and I wanted to avoid a mix of 3 channels, `defaults`, `odm2`, and `conda-forge`, in the future, when `odm2` packages move to `conda-forge`,  we can simplify this setup.